src/services/stellar/index.tsx

```diff
@@ -188,7 +191,7 @@ async function createOfframpAndMergeTransaction(
       throw new Error(`Unexpected offramp memo type: ${memoType}`);
   }
 
-  const stellarAsset = new Asset(code.string, issuer.stellarEncoding);
+  const stellarAsset = new Asset(code.string.replace('\0', ''), issuer.stellarEncoding);
```

Comment:
If we're using string.replace('\0', '') or assetCode.replace('\0', '') in multiple places, perhaps we should extract it into a separate function. This would enhance both readability and reusability. What do you think?